thoth/package_extract/image.py

```diff
@@ -178,6 +179,24 @@ def _parse_deb_dependency_line(line_str: str) -> typing.List[tuple]:
     return result
 
 
+def _gather_digests(path: str) -> typing.List[dict]:
+    """Calculate checksum for all files inside image."""
+    digests = []
+    for root, dirs, files in os.walk(path):
+        for file_ in files:
+            if file_.endswith('.py'):
+                filepath = os.path.join(root, file_)
+                if os.path.isfile(filepath):
+                    digest = hashlib.sha256()
+                    with open(filepath, 'rb') as afile:
+                        digest.update(afile.read())
+                    digests.append({
+                        'filepath': filepath,
```

Comment:
Let's have paths absolute to the analyzed image, not to point to a temporary extracted container image on host:

```
                        'filepath': filepath[len(path):],
```